tests/e2e.test.js

```diff
@@ -0,0 +1,13 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test.describe('SplashScreen', () => {
```

Comment:
@dnenov This needs update I think

Reply:
Yes, just updated it and added the playwright config which should handle the server start in the pipeline. Working through the failures one by one, but getting there, @QilongTang !

Reply:
OK, all passed now, the build is working!